packages/site/site-nextjs/src/sitePreview/appRouter/sitePreviewRoute.ts

```diff
@@ -40,14 +40,19 @@ export async function sitePreviewRoute(request: NextRequest) {
  * @param options.skipDraftModeCheck Allows skipping the draft mode check, only required when called from middleware.ts (see https://github.com/vercel/next.js/issues/52080)
  * @return If SitePreview is active the current preview settings
  */
-export async function previewParams(options: { skipDraftModeCheck: boolean } = { skipDraftModeCheck: false }): Promise<SitePreviewParams | null> {
+export async function previewParams(options: { skipDraftModeCheck: boolean } = { skipDraftModeCheck: false }): Promise<PreviewParams | null> {
     if (!options.skipDraftModeCheck) {
         if (!draftMode().isEnabled) return null;
     }
 
-    const cookie = cookies().get("__comet_preview");
-    if (cookie) {
-        return verifySitePreviewJwt(cookie.value);
+    const sitePreviewCookie = cookies().get("__comet_preview");
+    if (sitePreviewCookie) {
+        return verifyPreviewJwt(sitePreviewCookie.value);
```

Comment:
Contd: Why is `verifyPreviewJwt` used here and not `verifySitePreviewJwt`? Is `verifySitePreviewJwt` still needed?